### DISCUSSION TITLE
NLB with multiple open ports?

### DISCUSSION THREAD

**Discussion by dragosbulugean** [aws/copilot-cli#3589] [Q&A]
Hi,

We've been trying to forward 80 and 443 through NLB to our containers (they need both ports to generate customer certificates).

Is there any way to forward both?

**Comment by iamhopaul123** [comment]
Hello @dragosbulugean. Sorry this is not possible right now. Would you mind to cut us a GH issue with your use case? (Would be great to have more context!)

**Comment by dragosbulugean** [comment]
Hi @iamhopaul123 ,

Thanks for the response. Here's the issue, just woke up and having it hard to express some thoughts, but hope it clears it up: https://github.com/aws/copilot-cli/issues/3594
